trio/_core/_run.py

```diff
@@ -1254,6 +1261,13 @@ def run(
     GLOBAL_RUN_CONTEXT.runner = runner
     locals()[LOCALS_KEY_KI_PROTECTION_ENABLED] = True
 
+    if use_watchdog:
+        watchdog = TrioWatchdog()
+    else:
+        watchdog = None
+
+    GLOBAL_RUN_CONTEXT.watchdog = watchdog
```

Comment:
It doesn't really matter, but... any reason in particular this isn't an attribute on `runner`?

Reply:
No paticular reason - just where I put it at first.